extras/spoof.md

```diff
@@ -47,6 +47,38 @@ With this, navigate towards `Root -> DeviceProperties -> Add` and add your PCI r
 
 ![](../images/extras/spoof-md/config-gpu.png)
 
+For disabling NVIDIA Kepler you can use these device prperties:
+
+```
+<key>DeviceProperties</key>
+	<dict>
+    <key>PciRoot(0x0)/Pci(0x7,0x0)/Pci(0x0,0x0)/Pci(0x8,0x0)/Pci(0x0,0x0)</key>
```

Comment:
Does this not vary per hardware?

Reply:
Yes. It disables only NVIDIA in a specific slot, which can be found by gfxutil, Hackintool, OpenCore Configurator etc.